### DISCUSSION TITLE
Azure services for students

### DISCUSSION THREAD

**Discussion by MohamedElshreef1011** [community/community#54479] [GitHub Education]
### Hi everyone,

**Hi Guys, I am a university student. I entered the university mail in GitHub to get Azure services for students, and I sent them official documents proving that I am a student, but no one contacted me, even if I could get Azure services for students.**

**Comment by samus-aran** [comment]
Hi @MohamedElshreef1011 

You must be a verified student to receive GitHub Global Campus Benefits including the Azure services. Once you have been verified you can redeem your Azure benefits via the [Azure verification page](https://signup.azure.com/studentverification?offerType=1). If this does not work then we would suggest you reach out to [Azure Support](https://support.microsoft.com/en-us/topic/2315e669-8b1f-493b-5fb1-d88a8736ffe4) if you are having problems redeeming your Azure benefits as a verified GitHub Global Campus student.
